### ISSUE TITLE
Did fix for #2019 make it into master 3.0.2970? Still getting the error 

### ISSUE THREAD

**Issue by kdizzay** [tidusjar/Ombi#2051]
#### Ombi build Version:

V 3.0.2970

#### Update Branch:

master

#### Media Sever:

Plex

#### Operating System:

Ubuntu 16.04

#### Ombi Applicable Logs (from `/logs/` directory or the Admin page):

```
Mar 07 21:47:14 killinit Ombi[7862]:       The episode title Optimal Tip-to-Tip Efficiency we cannot find the parent series. The episode grandparentKey = 18680, grandparentTitle = Silicon Valley
Mar 07 21:47:14 killinit Ombi[7862]: warn: Ombi.Schedule.Jobs.Plex.PlexEpisodeSync[0]
Mar 07 21:47:14 killinit Ombi[7862]:       The episode title Sand Hill Shuffle we cannot find the parent series. The episode grandparentKey = 18680, grandparentTitle = Silicon Valley
Mar 07 21:47:14 killinit Ombi[7862]: warn: Ombi.Schedule.Jobs.Plex.PlexEpisodeSync[0]
Mar 07 21:47:14 killinit Ombi[7862]:       The episode title Runaway Devaluation we cannot find the parent series. The episode grandparentKey = 18680, grandparentTitle = Silicon Valley
Mar 07 21:47:14 killinit Ombi[7862]: warn: Ombi.Schedule.Jobs.Plex.PlexEpisodeSync[0]
Mar 07 21:47:14 killinit Ombi[7862]:       The episode title Bad Money we cannot find the parent series. The episode grandparentKey = 18680, grandparentTitle = Silicon Valley
Mar 07 21:47:14 killinit Ombi[7862]: warn: Ombi.Schedule.Jobs.Plex.PlexEpisodeSync[0]


```

#### Problem Description:

Same as issue #2019 

#### Reproduction Steps:

Please include any steps to reproduce the issue, this the request that is causing the problem etc.

Run Plex cacher and this is happening for all TV shows. 

Steps taken : 

1) Updated to 3.0.2970 from 3.0.2960
2) Refresh metadata of the entire TV shows library in Plex then run the cacher again. 

Issue still persists - do I need to remove/purge my existing db or something?

**Comment by tidusjar** [comment]
Yes the fix is in master. You can clear out your Plex content and Plex episodes tables

**Comment by kdizzay** [comment]
@tidusjar 

Sorry, how do I clear the tables? Delete ombi.db and schedule.db or is there a command I can use to purge the db?

**Comment by tidusjar** [comment]
You can delete it via sqlite3 using the command `Delete from PlexServerContent;` and `Delete from PlexEpisodes`

**Comment by kdizzay** [comment]
That did it - thanks, issue is no longer seen.

**Comment by jeffp1717** [comment]
For someone who might not know how to access sqlite3 to enter those commands, how would I fix this issue? Still persisting for me on 3.0.3173. It won’t scan in any of my tv.
